.pre-commit-config.yaml

```diff
@@ -7,6 +7,8 @@ repos:
     rev: v2.2.3  # Use the ref you want to point at
     hooks:
     - id: mixed-line-ending
+      args:
```

Comment:
tl;dr: remove this we handle it in the CI

We patch this config file in the CI so that we won't have mixed ending merged. However we don't enforce that in this config file given it causes issue on Windows: On Windows git can be configured to checkout file with \r\n, but to convert them back into \n when doing the commit, however this doesn't work with pre-commit given it is run on the file on the disk.

Reply:
How can I tell to my pre-commit ?

Reply:
nvm